src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int i = new Random().nextInt(Color.values().length);

```

Comment:
Make Random a class-level variable.
```suggestion
        int index = random.nextInt(Color.values().length);

```

Reply:
not fixed